src/View/TwigView.php

```diff
@@ -113,22 +113,17 @@ public function setTwig(Environment $twig): void
      */
     protected function resolveConfig(): array
     {
-        $charset = 'utf-8';
-        if (Configure::read('App.encoding') !== null) {
-            $charset = strtolower(Configure::read('App.encoding'));
```

Comment:
The `strtolower()` was unnecessary. Twig's [Environment](https://github.com/twigphp/Twig/blob/3.x/src/Environment.php#L104) uses upper cased string.